### DISCUSSION TITLE
Emular PC mais fraco para testar site

### DISCUSSION THREAD

**Discussion by diegoemerique** [frontendbr/forum#298] [Perguntas e Respostas]
Olá, não sei se a pergunta não está bem formulada mas vou explicar a situação.

Eu fiz um site que tem um vídeo de background, e por cima, a partir do scroll, entra outras seções do site com transitions e transforms.

O problema é que quando eu passei para alguns amigos, eles alegaram que estava travando muito. Meu pc é um i7 com 16gb de RAM. Aqui estava rodando liso, mas para eles, as animações estavam travando. Foi ai que eu tirei o vídeo, e para eles começou a rodar liso. 

Então eu deixo a dúvida. Qual seria a maneira mais eficaz de testa e encontrar problemas a performance do site? (alguma ferramenta, técnica, ou estudo necessário para isso);

**Comment by Lucasktrindade** [comment]
Fala cara,

Eu costumo testar a velocidade de carregamento dos sites que faço pelo
PageSpeed Insights, lá mostra o carregamento do site e a experiência do
usuário no mobile.
Link do site: https://developers.google.com/speed/pagespeed/insights/

**Comment by robsonbittencourt** [comment]
Talvez isso possa te ajudar.

https://twitter.com/addyosmani/status/767730617932840960

**Comment by kazzkiq** [comment]
Fazem algumas semanas que o Google Chrome [adicionou uma opção chamada CPU Throttling](https://plus.google.com/+AddyOsmani/posts/NRsAqshb17n) que serve justamente para "simular" dispositivos mais lentos.

<img width="531" alt="captura de tela 2016-09-05 as 19 57 03" src="https://cloud.githubusercontent.com/assets/1953194/18257855/10687916-73a1-11e6-9e04-5f33f9c854d2.png">

Você pode testar essa funcionalidade baixando o [Chrome Canary](https://www.google.com.br/chrome/browser/canary.html).

---

Edit: @robsonbittencourt foi mais rápido que eu! 😄

**Comment by diegoemerique** [comment]
Muito boa essa opção! 

Alguém conhece algum conteúdo que ensina a ler esses gráficos todos? :/

**Comment by barbier** [comment]
Você pode subir uma VM pelo VirtualBox. Configura +/- o pc dos seus colegas e experimenta

**Comment by felipefialho** [comment]
@barbier Acredito ser a melhor alternativa :)

**Comment by diegoemerique** [comment]
@barbier é acho que o VM é mais fácil para ver o que ta zoando ou não ;)
